src/components/FeatureAttributes/fields/FeatureAttributeReservedBoundingModeField/FeatureAttributeReservedBoundingModeField.tsx

```diff
@@ -37,7 +37,8 @@ export const FeatureAttributeReservedBoundingModeField: FC<
   if (
     !featureType ||
     !allowedFeatureTypes.includes(featureType) ||
-    !allowedDataTypes.includes(dataType)
+    !allowedDataTypes.includes(dataType) ||
+    !purposes.includes("synthesis")
```

Comment:
This is just the bounds of numeric features right? Definately not limited to synthesis

Reply:
Marked BoundingMode as core